testing/scenario_app/run_ios_tests.sh

```diff
@@ -41,25 +41,57 @@ fi
 # Can also be set via Simulator app Device > Rotate Device Automatically
 defaults write com.apple.iphonesimulator RotateWindowWhenSignaledByGuest -int 1
 
-cd $SRC_DIR/out/$FLUTTER_ENGINE/scenario_app/Scenarios
+SCENARIO_PATH=$SRC_DIR/out/$FLUTTER_ENGINE/scenario_app/Scenarios
+cd $SCENARIO_PATH
+
+RESULT_BUNDLE_FOLDER="ios_scenario_xcresult"
+RESULT_BUNDLE_PATH="${SCENARIO_PATH}/${RESULT_BUNDLE_FOLDER}"
+
+# Zip and upload xcresult to luci.
+# First parameter ($1) is the zip output name.
+ZIP_AND_UPLOAD_XCRESULT_TO_LUCI () {
+  # Using absolute directory causes the zip containing all the sub directories.
```

Comment:
Can you clear this up please?  You are probably running into the difference between having a trailing slash or not fwiw.